test-runner-frontend/src/Views/Report.js

```diff
@@ -0,0 +1,390 @@
+                    {item[0]}
+                  </HashLink>
+                </Typography>
+              ))}
+            </Box>
+
+            {/* Charts BOX */}
+            <Box sx={{ width: '100%', marginLeft: '15%', display: 'flex', flexDirection: 'row' }}>
+              {/* Bar columns BOX */}
+
+              <Box sx={{ width: '100%', display: 'flex', flexDirection: 'column' }}>
+                {classesAndBenchmarksState.map((benchmarkData, i) => {
+                  var currentChart = benchmarkData;
+                  var nextChart = i <= classesAndBenchmarksState.length ? classesAndBenchmarksState[i + 1] : null;
+                  if (i % 2 === 0 || i === 0) {
+                    return (
+                      <Box key={i} sx={{ width: '100%', display: 'flex', flexDirection: 'row' }}>
+                        <Box key={i} sx={{ marginBottom: '3.6%', marginRight: '10%' }}>
+                          <Typography variant="h5" id={`${currentChart[0]}`} sx={{ display: 'block' }}>
+                            <b>{currentChart[0]}</b>
+                          </Typography>
+                          <IconButton
+                            style={{ float: 'right' }}
+                            edge="start"
+                            color="inherit"
+                            onClick={() => handleClickOpenDialog(currentChart[1])}
+                            aria-label="open"
+                          >
+                            <MoreHorizIcon />
+                          </IconButton>
+
+                          <BarCharts classesAndBenchmarksState={currentChart[1]} />
+                        </Box>
+
+                        {nextChart ? (
+                          <Box key={i + 1} sx={{ marginBottom: '3%' }}>
+                            <Typography variant="h5" id={`${nextChart[0]}`} sx={{ display: 'block' }}>
+                              <b>{nextChart[0]}</b>
+                            </Typography>
+                            <IconButton
+                              style={{ float: 'right' }}
+                              edge="start"
+                              color="inherit"
+                              onClick={() => handleClickOpenDialog(nextChart[1])}
+                              aria-label="open"
+                            >
+                              <MoreHorizIcon />
+                            </IconButton>
+                            <BarCharts classesAndBenchmarksState={nextChart[1]} />
+                          </Box>
+                        ) : null}
+                      </Box>
+                    );
+                  }
+                  return null;
+                })}
+              </Box>
+            </Box>
+
+            <Snackbar
+              open={openSnackBar}
+              onClose={handleCloseSnackBar}
+              autoHideDuration={snackBarMessageDuration}
+              TransitionComponent={transitionSnackBar}
```

Comment:
is `TransitionComponent` always `TransitionSnackbar`?

Reply:
Yes, I used the same snackbar and just updated the message, is that ok?

Reply:
do we need to pass `transition` through to the state anymore then?

Reply:
Oh that's true, we don't.
Done.